autosklearn/evaluation/test_evaluator.py

```diff
@@ -129,7 +131,7 @@ def eval_t(
     evaluator = TestEvaluator(
         configuration=config,
         backend=backend,
-        metric=metric,
+        metric=metrics,
```

Comment:
Can't we use `metrics` throughout the whole code?

Reply:
That was an oversight by me, sorry.